service/matching/physical_task_queue_manager.go

```diff
@@ -108,6 +114,7 @@ type (
 		UnloadFromPartitionManager()
 		String() string
 		QueueKey() *PhysicalTaskQueueKey
+		Matcher() *TaskMatcher
```

Comment:
what's this for? I don't think we want the matcher to be public, it's an implementation detail

Reply:
Sorry, this method was added for an older idea but is no longer relevant. Removed.